it/uk/gov/hmrc/incorporatedentityidentificationfrontend/repositories/JourneyConfigRepositoryISpec.scala

```diff
@@ -47,18 +47,6 @@ class JourneyConfigRepositoryISpec extends ComponentSpecHelper with AbstractPati
       await(repo.count) mustBe 1
     }
 
-    "successfully find journeyConfig" when {
```

Comment:
Guess below test is missed to put back

"successfully insert journeyConfig" in {
      await(repo.insertJourneyConfig(testJourneyId, testInternalId, testLimitedCompanyJourneyConfig))
      await(repo.findJourneyConfig(testJourneyId, testInternalId)) must contain(testLimitedCompanyJourneyConfig)
    }